ckcp/openshift_dev_setup.sh

```diff
@@ -211,18 +211,24 @@ install_cert_manager(){
 
 check_cert_manager() {
   # perform a dry-run create of a cert-manager
-  # Certificate resource in order to verify that CRDs are installed and all the
-  # required webhooks are reachable by the K8S API server.
+  # Certificate resource in order to verify that the cert-manager CRDs are installed on the Kubernetes API server
   local i=0
   until kubectl create -f "$CKCP_DIR/openshift/base/certs.yaml" --dry-run=client >/dev/null 2>&1; do
```

Comment:
I thought it had been fixed, maybe in another inflight PR. `>/dev/null 2>&1;`is not a good practice

Reply:
Also I find the check a bit awkward. Why not simply asking the API server whether the crd is available rather than trying to create a resource and seeing the creation failing?

Reply:
1. I find the check logic from [here](https://github.com/cert-manager/cert-manager/blob/release-1.9/pkg/util/cmapichecker/cmapichecker.go#L87) 
2. I wanna ensure all the resources defined in `$CKCP_DIR/openshift/base/certs.yaml` are ready, so I directly apply the `certs.yaml` file. I think all of the resources need time to be installed in the API server. 
3.with `--dry-run=client`, it doesn't need to a cleanup 

please correct me if my understanding is not correct.

Reply:
1. and 2. still awkward
3. if you ask the API whether the resource exists it does not need any cleanup either

Really it increases readability and really checks what is described in the comment.
There are potentially other reasons for the creation of a resource not being successful besides the availability of the API, e.g. the successful execution of admission webhooks and other policies (I don't know whether they are in use with cert-manager). I am ok for you to test them (if they are used) but then the comment needs to be changed.
Otherwise kubectl get crd and potentially checking that versions match seem a more straightforward approach.